packages/as-types/assembly/amount.ts

```diff
@@ -104,6 +104,9 @@ export class Amount {
 
   /**
    * Updates Args with current currency serialized.
+   *
+   * @deprecated implements `Serializable`
```

Comment:
I guess it's too soon to do that. You say it's deprecated without proposing alternative ...

Reply:
so I will implement the alternative ^^

Reply:
I created issue #91 to do it later, and removed @deprecated